gammapy/estimators/parameter.py

```diff
@@ -330,4 +358,19 @@ def run(self, datasets, parameter):
                 result.update(self.estimate_scan(datasets, parameter))
 
         result.update(self.estimate_counts(datasets))
+
+        result["fit_status"] = int(result["success"])
+        if result["fit_status"] > 0 and "success_err" in result:
+            result["fit_status"] += int(result["success_err"])
+            if "scan" in self.selection_optional and "success_scan" in result:
+                if False in result["success_scan"]:
+                    result["fit_status"] = 1
```

Comment:
see comment above, if I get it correctly `success_err` is False `if False in result["success_scan"]` so this is unnecessary.